notebooks/3_KernelMethods.ipynb

```diff
@@ -638,7 +664,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Even small regularizations lead to an increase in accuracy for predicting out-of-sample data. The best regularization here is {{round(best_regularization, 8)}}."
+    "Even small regularizations lead to an increase in accuracy for predicting out-of-sample data. The best regularization here is `round(best_regularization, 8)`."
```

Comment:
same comment as above.

Reply:
Again, this doesn't work on my Jupyter notebook.